daal4py/mb/gbt_convertors.py

```diff
@@ -567,6 +571,22 @@ def get_gbt_model_from_catboost(booster: Any) -> Any:
             "Categorical features are not supported in daal4py Gradient Boosting Trees"
         )
 
+    objective = booster.get_params().get("objective")
```

Comment:
```suggestion
    objective = booster.get_params().get("objective", "")
```

Otherwise the checks below might fail, no?

```bash
>>> "A" in None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'NoneType' is not iterable
```